examples/cli/discovery.md

```diff
@@ -124,7 +124,7 @@ For example, if you run this command:
 benchit benchmark scripts/max_depth.py
 ```
 
-You will get a result that looks very similar to the [Hello World tutorial](#hello-world) tutorial. However, if you peek into `max_depth.py`, you can see that there are two `torch.nn.Linear` modules that make up the top-level model.
```

Comment:
Line 113  See the [Benchmark Multiple Scripts documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#benchmark-multiple-scripts) for more details.

Reply:
got it!